tools/tensorflow_docs/api_generator/generate_lib.py

```diff
@@ -167,7 +167,7 @@ def title(self):
   @property
   def children(self):
     return sorted(
-        self._children, key=lambda x: (x.full_name.upper(), x.full_name))
+        self._children, key=lambda x: x.full_name)
```

Comment:
Can you put this line on the same line as `return sorted`? (Google's linter will complain if you don't do that)

So 

`return sorted(self._children, key=lambda x: x.full_name)` 

instead of 

```
return sorted(
    self._children, key=lambda x: x.full_name)
```